themes/app/templates/Includes/Footer.ss

```diff
@@ -1 +1,47 @@
-<footer>Irsan Arisandy @ SilverStripe &copy; $Now.Year</footer>
+<footer>
+    <div class="container footer__partners">
+        <div class="row">
+            <div class="col-xl-2 col-lg-2 col-md-3 col-sm-6 col-6 ">
+                Our Partners
+            </div>
+        </div>
+        <div class="row">
+            <div class="col-xl-2 col-lg-2 col-md-3 col-sm-6 col-6">
+                    $SiteConfig.SponsorLogo1
+            </div>
+            <div class="col-xl-2 col-lg-2 col-md-3 col-sm-6 col-6">
+                    $SiteConfig.SponsorLogo2
+            </div>
+            <div class="col-xl-2 col-lg-2 col-md-3 col-sm-6 col-6">
+                    $SiteConfig.SponsorLogo3
+            </div>
+            <div class="col-xl-2 col-lg-2 col-md-3 col-sm-6 col-6">
+                    $SiteConfig.SponsorLogo4
+            </div>
+            <div class="col-xl-2 col-lg-2 col-md-3 col-sm-6 col-6">
+                    $SiteConfig.SponsorLogo5
+            </div>
+            <div class="col-xl-2 col-lg-2 col-md-3 col-sm-6 col-6">
+                    $SiteConfig.SponsorLogo6
+            </div>
+        </div>
+    </div>
+    <div class="container-fluid footer__info">
+        <div class="row">
+            <div class="col-xl-3 col-lg-3 col-md-6 col-sm-12">
```

Comment:
Similar concept here as above with the classes, i think you can get rid of `col-xl-3` :)